app/helpers/view_helper.rb

```diff
@@ -0,0 +1,7 @@
+module ViewHelper
+
+  def title_with_error_prefix(title, error:)
+    "#{"Error: " if error}#{title}"
+  end
```

Comment:
Definitely room for this in the govuk-components helpers. Every service will need something like it.

Reply:
Yeah, I wasn't sure whether it'd be worth making as component or not, as it's such a small amount of code. Maybe?